lib/stat_tracker.rb

```diff
@@ -10,7 +10,7 @@ def initialize(locations)
     load_managers(locations)
   end
 
-  def self.from_csv(locations = {games: './data/games_sample.csv', teams: './data/teams_sample.csv', game_teams: './data/game_teams_sample.csv'})
+  def self.from_csv(locations = {games: './data/games.csv', teams: './data/teams_sample.csv', game_teams: './data/game_teams_sample.csv'})
```

Comment:
Why run the full games array? That's gonna be *really slow* to test with

Reply:
Good catch, sorry.  Was validating that i got correct spec answer.  Moved back to games_sample